client/app/components/packages/rwcds-form/show.hbs

```diff
@@ -0,0 +1,249 @@
+          Has the project changed since submission of the Pre-Application Statement (PAS)?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.pasForm.dcpHasprojectchangedsincesubmissionofthepas "Yes" "No"}}
+        </span>
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Project site description:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpProjectsitedescription}}
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Proposed development description:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpProposedprojectdevelopmentdescription}}
+      </p>
+
+      <p class="grid-x ruled-top">
+        <strong class="cell auto large-padding-right">
+          Will construction be done in multiple phases?
+        </strong>
+        <span class="cell large-3">
+          {{#if (eq @package.rwcdsForm.dcpConstructionphasing 717170000)}}
+            Single
+          {{else if (eq @package.rwcdsForm.dcpConstructionphasing 717170001)}}
+            Multiple
+          {{else}}
+            Not Applicable
+          {{/if}}
+        </span>
+      </p>
+
+      <p class="grid-x ruled-top">
+        <strong class="cell auto large-padding-right">
+          What is the proposed analysis year of the proposed action?
+        </strong>
+        <span class="cell large-3">
+          {{@package.rwcdsForm.dcpBuildyear}}
+        </span>
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Analysis year rationale:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpRationalbehindthebuildyear}}
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          History of sites within the Project Area:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpSitehistory}}
+      </p>
+    </section>
+
+    <section class="form-section">
+      <h2 class="section-header">
+        <span id="" class="section-anchor"></span>
+        Proposed Actions
+      </h2>
+
+      {{!-- TODO: add list of actions --}}
+
+      <p class="ruled-top">
+        <strong>
+          Purpose and need for all proposed actions:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpPurposeandneedfortheproposedaction}}
+      </p>
+
+      <p class="grid-x ruled-top">
+        <strong class="cell auto large-padding-right">
+          Does the applicant plan to develop a 100% affordable housing development?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.pasForm.dcpIsplannigondevelopingaffordablehousing "Yes" "No"}}
+        </span>
+      </p>
+
+      <p class="grid-x ruled-top">
+        <strong class="cell auto large-padding-right">
+          Is the applicant seeking actions from other City/State/Federal agencies?
+        </strong>
+        <span class="cell large-3">
+          {{if (eq @package.rwcdsForm.dcpIsapplicantseekingaction 717170000) "Yes"}}
```

Comment:
The optionset helper can be used here, too. 
This time, you'll have to 
1) add the optionset export to `app/models/rwcds-form.js`, because it isn't already in there. e.g.

```
export const DCPISAPPLICANTSEEKINGACTION_OPTIONSET = {
  YES: {
    code: true,
    label: 'Yes',
  },
  NO: {
    code: false,
    label: 'No',
  },
};
```

2) register it in `helpers/optionset.js`, following the other examples. e.g.
```
  rwcdsForm: {
    dcpHasprojectchangedsincesubmissionofthepas: DCPHASPROJECTCHANGEDSINCESUBMISSIONOFTHEPAS_OPTIONSET,
    dcpConstructionphasing: DCPCONSTRUCTIONPHASING_OPTIONSET,
    dcpIsapplicantseekingaction: DCPISAPPLICANTSEEKINGACTION_OPTIONSET,
  },
```